src/Trains.NET.Rendering/Game/Game.cs

```diff
@@ -118,8 +118,6 @@ public void Render(ICanvas canvas)
                 return;
             }
 
-            canvas.Clear(Colors.White);
```

Comment:
Please put this back, without it when you turn the terrain layer off everything goes wonky :(

Reply:
I wish there was like a toggle between terrain and this. It is such a waste to clear the screen if we just draw on top of it all.

Reply:
This can be achieved..

Reply:
Added clear back in for now, don't want to break things :)